lib/urlutils.ts

```diff
@@ -0,0 +1,9 @@
+const deliveryDomain = (process.env.NEXT_PUBLIC_MUX_BYO_DOMAIN) ? process.env.NEXT_PUBLIC_MUX_BYO_DOMAIN : "mux.com"
```

Comment:
This is a good opportunity to just use the [new JS hotness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). This can be:

```javascript
const deliveryDomain = process.env.NEXT_PUBLIC_MUX_BYO_DOMAIN ?? "mux.com";
```

Reply:
I would recommend `||` here--it's possible that that env var might be set to an empty string and `??` will pass it along.

Reply:
🔥🔥🔥  I used `||` in the end, but this is great, thanks!